diagnostics.yml

```diff
@@ -351,6 +347,24 @@ extends:
           enableNugetValidation: true
           symbolPublishingAdditionalParameters: '/p:PublishSpecialClrFiles=false'
           publishInstallersAndChecksums: true
+          SDLValidationParameters:
+            enable: false
```

Comment:
@hoyosjs, do we want to enable the SDK validation? I left it enabled in the symuploader 1ES changes.

Reply:
Given the bit marshalling we do, all good with disabling it. 1es doesn't crack archives like ours did, but when we upload on a per-leg basis we get that coverage.